tests/testthat/test-derive_extreme_records.R

```diff
@@ -1,22 +1,22 @@
 # derive_extreme_records ----
```

Comment:
Could we add a test for this if/else? 

![image](https://user-images.githubusercontent.com/10111024/236567644-ab860a1d-61a2-4ae6-847c-eecb5e9c77f2.png)

Reply:
Done: https://github.com/pharmaverse/admiral/issues/1896